opslevel/resource_opslevel_service.go

```diff
@@ -327,12 +338,16 @@ func (r *ServiceResource) Update(ctx context.Context, req resource.UpdateRequest
 		LifecycleAlias: NullableStringConfigValue(planModel.LifecycleAlias),
 		Name:           opslevel.NewNullableFrom(planModel.Name.ValueString()),
 		OwnerInput:     opslevel.NewIdentifier(),
+		Parent:         opslevel.NewIdentifier(),
```

Comment:
shouldn't this only be set IF the end user has configured it in their plan?

Reply:
The API won't accept an empty string here so I added a validator to this field. That leaves the following:
- `parent = <value>` the value is set
- `parent = null` no parent is set on create, or the parent is unset on update
- `// parent = <value` same as `null`

Reply:
Did you test my edgecase?  ```where its always unsetting the value if its set via the UI but not actually in the Terraform config```